tfexec/terraform.go

```diff
@@ -125,8 +130,27 @@ func (tf *Terraform) SetLogPath(path string) error {
 
 // SetAppendUserAgent sets the TF_APPEND_USER_AGENT environment variable for
 // Terraform CLI execution.
-func (tf *Terraform) SetAppendUserAgent(ua string) {
+func (tf *Terraform) SetAppendUserAgent(ua string) error {
 	tf.appendUserAgent = ua
+	return nil
+}
+
+// SetDisablePluginTls sets the TF_DISABLE_PLUGIN_TLS environment variable for
+// Terraform CLI execution.
+func (tf *Terraform) SetDisablePluginTls(disabled bool) error {
```

Comment:
initialism: https://github.com/golang/go/wiki/CodeReviewComments#initialisms
```suggestion
func (tf *Terraform) SetDisablePluginTLS(disabled bool) error {
```